### PR TITLE
Fix broken url for testing instructions.

### PR DESCRIPTION
**Pull request by RafayGhafoor**
----

I've completed each of the following or determined they are not applicable:

- [X] Made a plan to communicate any major developer interface changes (or N/A)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @RafayGhafoor!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by RafayGhafoor** [comment]
@robrap, Please review.

File: docs/testing_and_debugging.rst [hunk 1]
```diff
@@ -96,4 +96,4 @@ Most tests are run in Firefox by default.  To use Chrome for tests that normally
 use Firefox instead, prefix the test command with
 ``SELENIUM_BROWSER=chrome SELENIUM_HOST=edx.devstack.chrome``.
 
-.. _edx-platform testing documentation: https://github.com/openedx/edx-platform/blob/master/docs/guides/testing/testing.rst#running-python-unit-tests
+.. _edx-platform testing documentation: https://github.com/openedx/edx-platform/blob/master/docs/concepts/testing/testing.rst#running-python-unit-tests
```
Comment (robrap) [inline docs/testing_and_debugging.rst]: Thanks. We should probably update to https://docs.openedx.org/projects/edx-platform/en/latest/concepts/testing/testing.html#running-python-unit-tests now that these are published.

Comment (RafayGhafoor) [reply docs/testing_and_debugging.rst]: done

**Review by robrap — APPROVED** [review]
Thank you!

**Comment by RafayGhafoor** [comment]
@robrap, I don't have the option to merge this PR, can you please do it. Thank you.

**Comment by openedx-webhooks** [comment]
@RafayGhafoor 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/devstack&entry.867055334=https://github.com/openedx/devstack/pull/1208&entry.1484655318=https://github.com/RafayGhafoor&entry.752974735=2023-10-14+20:38&entry.1917517419=2023-10-16+16:32&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by robrap** [comment]
Thanks again @RafayGhafoor. I just needed to wait for tests to complete. Note that I squashed using the github UI, but otherwise that would be something you might need to do on other PRs.

**Comment by RafayGhafoor** [comment]
yes, I understand @robrap. I also made this PR from the UI and it wasn't allowing me to squash since I am not a maintainer I guess.

**Comment by robrap** [comment]
I think you can only "Squash and Merge" through the UI. So otherwise, you'd need to get your fork locally and squash and push the changes back up.

**Pull request merged**